concent_api/delete-unsupported-messages.py

```diff
@@ -0,0 +1,40 @@
+import django
+import os
+import sys
+
+from logging import getLogger
+
+from common.helpers import deserialize_message
+
+from golem_messages.exceptions import MessageError
+
+sys.path.append('concent_api')
+os.environ['DJANGO_SETTINGS_MODULE'] = "concent_api.settings"
```

Comment:
Why is this not a `Command` anymore?

Reply:
We will not be merging it into master. The problem is that the code does not make sense in the release it would be merged into. It should always be executed on an older release (0.8.0 in this case) that still has the old schema and is useless on the current one. This would be confusing.

I have decided to create a standalone script and simply put it in release notes with instructions for the admin. See https://github.com/golemfactory/concent-deployment/issues/229

Reply:
It could for example read or generate Concent version and don't start if the version is below 0.8.0/